examples/kafka-streams-examples/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
latest for tests and 3.2.0 in examples. I will raise a ticket to fix this for all examples. #246